src/k8s-engine/lima.ts

```diff
@@ -1326,13 +1372,19 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
         }
 
         // We have no good estimate for the rest of the steps, go indeterminate.
-        timers.clearInterval(this.progressInterval);
-        this.progressInterval = undefined;
+        if (enabledK3s) {
+          timers.clearInterval(this.progressInterval as ReturnType<typeof timers.setInterval>);
+          this.progressInterval = undefined;
+        }
 
         if ((await this.status)?.status === 'Running') {
           this.lastCommandComment = 'Stopping existing instance';
           await this.progressTracker.action(this.lastCommandComment, 100, async() => {
-            await this.ssh('sudo', '/sbin/rc-service', 'k3s', 'stop');
+            try {
+              await this.ssh('sudo', '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+            } catch (err) {
+              console.log(`Failed to stop k3s: ${ err }`, err);
```

Comment:
We shouldn't be eating the exception here.

Reply:
I saw it trigger an exception, and thought it was because the service `k3s` hadn't been created yet when you start with a cleared cache, and with k8s off. But I can't repro it -- it's probably unlikely for `status` to be `'Running'` on this code path.